src/sourmash/cli/tax/classify.py

```diff
@@ -0,0 +1,64 @@
+"""classify genomes"""
+
+import sourmash
+from sourmash.logging import notify, print_results, error
+
+#https://stackoverflow.com/questions/12116685/how-can-i-require-my-python-scripts-argument-to-be-a-float-between-0-0-1-0-usin
```

Comment:
I think this might be a simpler approach (and also easier to read/understand in the future? =])
https://stackoverflow.com/questions/55324449/how-to-specify-a-minimum-or-maximum-float-value-with-argparse#55410582